components/dashboard/src/start/StartWorkspace.tsx

```diff
@@ -194,17 +193,39 @@ export default class StartWorkspace extends React.Component<StartWorkspaceProps,
       // When in this state, we expect it to become running or stopping anytime soon.
       case "interrupted":
         phase = StartPhase.Running;
-        statusMessage = <p className="text-base text-gray-400">Checking On Workspace …</p>;
+        statusMessage = <p className="text-base text-gray-400">Checking workspace …</p>;
         break;
 
       // Stopping means that the workspace is currently shutting down. It could go to stopped every moment.
       case "stopping":
-        statusMessage = <p className="text-base text-gray-400">Stopping …</p>;
+        phase = StartPhase.Stopping;
+        statusMessage = <div>
+          <div className="flex space-x-3 items-center rounded-xl py-2 px-4 h-16 w-72 mt-4 bg-gray-100">
+            <div className="rounded-full w-3 h-3 text-sm bg-gitpod-kumquat">&nbsp;</div>
+            <div>
+              <p className="text-gray-700 font-semibold">{this.state.workspaceInstance.workspaceId}</p>
+              <p>{this.state.contextUrl}</p>
```

Comment:
**issue**: Let's truncate this as it can break the layout for this component!

<img width="315" alt="Screenshot 2021-03-20 at 12 48 33 AM" src="https://user-images.githubusercontent.com/120486/111849387-23edb980-8916-11eb-842d-1bed90822758.png">

Reply:
Moved to https://github.com/gitpod-io/gitpod/issues/3507.